database/tables/pl_courses.pg

```diff
@@ -5,6 +5,7 @@ columns
     display_timezone: text
     example_course: boolean not null default false
     grading_queue: text
+    help_link: boolean default false
```

Comment:
Let's make this `not null`.

Reply:
Done in [719cadc](https://github.com/PrairieLearn/PrairieLearn/pull/6141/commits/719cadc207a33807aa963181de3eaae16ed56c74)